src/explore-education-statistics-admin/src/pages/release/data/components/DataFileReplacementPlan.tsx

```diff
@@ -61,7 +61,12 @@ const DataFileReplacementPlan = ({
     error,
     setState: setPlan,
   } = useAsyncRetry(
-    () => dataReplacementService.getReplacementPlan(fileId, replacementFileId),
+    () =>
+      dataReplacementService.getReplacementPlan(
+        releaseId,
+        fileId,
+        replacementFileId,
+      ),
     [fileId],
```

Comment:
Ah, we're missing other dependencies in this array. Should be `[releaseId, fileId, replacementFileId]`

Reply:
Oops, I've added that now.